stories/Components/Forms/Dropdowns/Multiselect/MultiSelect.jsx

```diff
@@ -11,7 +11,6 @@ const SelectTag = ({ text, eleId, ...args }) => {
   return (
     <div className="multi-select" data-multi-select="">
       <button
-        className="select-control"
         aria-label="Region"
         aria-expanded="false"
         id={`select_button-${eleId}`}
```

Comment:
@shishirphadke we are using this ID for filter search bar. Because there are multiple buttons on same page and chips generate on run time. if you want to discuss we can discuss this on DSM.

Reply:
@shishirphadke Done.